src/components/IconTile.module.scss

```diff
@@ -53,10 +53,14 @@
   font-size: 0.8rem;
   color: rgba(134,134,155,0.6);
   width: 100%;
-  white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   text-align: center;
+  height: 30px;
+  word-break: break-all;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
```

Comment:
I like this design change. Can we avoid the fixed height and `-webkit` specific attributes?

Reply:
Firefox uses the `-webkit` prefix for this non-standard CSS feature: https://caniuse.com/?search=line-clamp

Reply:
Yeah, this feature doesn't exist outside of webkit, but I'll see if there's another way we could handle this to avoid it